mobility_games/games/grid_game.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+
```

Comment:
Can you turn all the Tabs in this file into 4 spaces?  There's a really easy way to do this in Sublime, let me know if you want me to show you!